src/main/scala/org/globalforestwatch/summarystats/carbonflux/CarbonFluxRDD.scala

```diff
@@ -1,9 +1,11 @@
 package org.globalforestwatch.summarystats.carbonflux
 
 import cats.implicits._
-import geotrellis.contrib.polygonal._
 import geotrellis.raster._
 import geotrellis.raster.rasterize.Rasterizer
+import geotrellis.raster.summary.GridVisitor
+import geotrellis.raster.summary.polygonal._
```

Comment:
Codacy found an issue: [Use brackets to group imports from the same package](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)